<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomatoClock</class>
 <widget class="QWidget" name="TomatoClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="soundalue_slider">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>200</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
	height: 5px;
	background: #e67e22;
}
QSlider::groove:vertical {
	width: 5px;
	background: #e67e22;
}
QSlider::handle:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
	border: 1px solid #5c5c5c;
	width: 14px;
	margin: -5px 0;
	border-radius: 7px;
}
QSlider::handle:vertical {
	background: qlineargradient(x1:1, y1:1, x2:0, y2:0, stop:0 #b4b4b4, stop:1 #8f8f8f);
	border: 1px solid #5c5c5c;
	height: 14px;
	margin: 0 -5px;
	border-radius: 7px;
}
QSlider::add-page:horizontal {
    background: white;
}
QSlider::add-page:vertical {
    background: white;
}
QSlider::sub-page:horizontal {
    background: #e67e22;
}
QSlider::sub-page:vertical {
    background: #e67e22;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>今日已专注：</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
   
	font:  &quot;Times New Roman&quot;;
    font-size: 16px;
    color: #d0dfe6;
    font-style: normal;
    font-weight: normal;
 
    border-style: solid;
    border-width: 1px;
    border-color: #134857;
    border-radius: 10px;
 
    padding-left: 4px;
    padding-top: 0px;
 
    background-color: #11659a;
    background-repeat: no-repeat;
    background-position: left center;
}</string>
      </property>
      <property name="text">
       <string>任务名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskNameLineEdit">
      <property name="styleSheet">
       <string notr="true">* {
	outline: none;
}

QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 3px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #F2F2F2; /* 背景颜色 */
	color: #A0A0A0; /* 文本颜色 */
	selection-background-color: #A0A0A0; /* 选中文本的背景颜色 */
	selection-color: #373834; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px solid #298DFF;
	border-radius: 3px;
	background-color: #F2F2F2;
	color: #298DFF;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}

QLineEdit[echoMode=&quot;2&quot;] { /* QLineEdit有输入掩码时的状态 */
	lineedit-password-character: 9679;
	lineedit-password-mask-delay: 2000;
}

QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 1px solid #CDCDCD;
	background-color: #CDCDCD;
	color: #B4B4B4;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	background-color: #CDCDCD;
	color: #F2F2F2;
}
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
   
	font:  &quot;Times New Roman&quot;;
    font-size: 18px;
    color: #d0dfe6;
    font-style: normal;
    font-weight: normal;
 
    border-style: solid;
    border-width: 1px;
    border-color: #134857;
    border-radius: 10px;
 
    padding-left: 4px;
    padding-top: 0px;
 
    background-color: #11659a;
    background-repeat: no-repeat;
    background-position: left center;
}</string>
      </property>
      <property name="text">
       <string>任务时长</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskDurationLineEdit">
      <property name="styleSheet">
       <string notr="true">* {
	outline: none;
}

QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 3px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #F2F2F2; /* 背景颜色 */
	color: #A0A0A0; /* 文本颜色 */
	selection-background-color: #A0A0A0; /* 选中文本的背景颜色 */
	selection-color: #373834; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px solid #298DFF;
	border-radius: 3px;
	background-color: #F2F2F2;
	color: #298DFF;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}

QLineEdit[echoMode=&quot;2&quot;] { /* QLineEdit有输入掩码时的状态 */
	lineedit-password-character: 9679;
	lineedit-password-mask-delay: 2000;
}

QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 1px solid #CDCDCD;
	background-color: #CDCDCD;
	color: #B4B4B4;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	background-color: #CDCDCD;
	color: #F2F2F2;
}
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
   
	font:  &quot;Times New Roman&quot;;
    font-size: 18px;
    color: #d0dfe6;
    font-style: normal;
    font-weight: normal;
 
    border-style: solid;
    border-width: 1px;
    border-color: #134857;
    border-radius: 10px;
 
    padding-left: 4px;
    padding-top: 0px;
 
    background-color: #11659a;
    background-repeat: no-repeat;
    background-position: left center;
}</string>
      </property>
      <property name="text">
       <string>休息时长</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskBreakTimeLineEdit">
      <property name="styleSheet">
       <string notr="true">* {
	outline: none;
}

QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 3px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #F2F2F2; /* 背景颜色 */
	color: #A0A0A0; /* 文本颜色 */
	selection-background-color: #A0A0A0; /* 选中文本的背景颜色 */
	selection-color: #373834; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px solid #298DFF;
	border-radius: 3px;
	background-color: #F2F2F2;
	color: #298DFF;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}

QLineEdit[echoMode=&quot;2&quot;] { /* QLineEdit有输入掩码时的状态 */
	lineedit-password-character: 9679;
	lineedit-password-mask-delay: 2000;
}

QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 1px solid #CDCDCD;
	background-color: #CDCDCD;
	color: #B4B4B4;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	background-color: #CDCDCD;
	color: #F2F2F2;
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="Timer">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>开始计时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseButton">
      <property name="text">
       <string>暂停计时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endButton">
      <property name="text">
       <string>结束计时</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="totaltime_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0分钟</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>481</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>任务名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任务时长</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>休息时长</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>默认任务1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>默认任务2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>45</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>默认任务3</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>65</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>15</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="addTaskBtn">
      <property name="text">
       <string>增加任务</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editTaskBtn">
      <property name="text">
       <string>编辑任务</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delTaskBtn">
      <property name="text">
       <string>删除任务</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
